ng-dev/misc/generated-files/update-generated-files.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license
+ * Copyright Google LLC
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ChildProcess} from '../../utils/child-process.js';
+import {Spinner} from '../../utils/spinner.js';
+import {BAZEL_BIN} from '../../utils/bazel-bin.js';
+import {green, Log} from '../../utils/logging.js';
+
+export async function updateGeneratedFileTargets(): Promise<void> {
+  const spinner = new Spinner('Querying for all generated file targets');
+
+  // Query for all of the generated file targets
+  const result = await ChildProcess.spawn(
+    BAZEL_BIN,
+    ['query', `"kind(nodejs_binary, //...) intersect attr(name, '.update$', //...)"`],
```

Comment:
Sometimes these things are called `.accept`. Maybe we should make this tag-based?

Reply:
I think I would prefer to expand the tooling to handle it. Because part of the goal here is that we capture anything that would fail in the tests.  Thoughts on that route?

Reply:
I'm just guessing that there could be all types of targets for updating checked-in files. E.g. it may even be called `//:update-some-file`. Those should be marked with `tags = [generated-file]` in their macros IMO

We can always expand this though; so not feeling strongly for now. I think it's cleaner though; as long as we have control over targets

Reply:
Lets keep it this way for now as we will have to deal with the fact that we don't have controls of the base macros in some cases for applying the tags, for example on `generated_file_test`, it comes from rules_nodejs but is actually itself a macro for both the update and the test action.

I will work on a follow up that expands what it does.  This will allow us to at least update and address updating the generated files on renovate PRs in framework.